README.md

```diff
@@ -9,7 +9,7 @@ The Health package provides a basic infrastructure that Swift applications can u
 As an application developer, you create an instance of the `Health` class and then register one or more health checks. A health check can be either a closure that conforms to the `HealthCheckClosure` typealias or a class that conforms to the `HealthCheck` protocol. Once you have your health checks in place, you can ask your `Health` instance for its status.
 
 ## Swift version
-The latest version of Health works with the `4.0` version of the Swift binaries. You can download this version of the Swift binaries by following this [link](https://swift.org/download/#snapshots).
+The latest version of Health works with the `4.2` version of the Swift binaries. You can download this version of the Swift binaries by following this [link](https://swift.org/download/#snapshots).
```

Comment:
Can this say 4.1.2 instead of 4.2? 4.2 isn't going to be out for a few months , it could also say something like works with 4.1.2 and later.